runbooks/OrphanedVirtualMachineImages.md

```diff
@@ -0,0 +1,77 @@
+# OrphanedVirtualMachineImages
+
+## Meaning
+
+This alert fires when a VMI (virt-launcher-* pod) is running on a node that does not have a running `virt-handler-*` pod.  
+
+## Impact
+
+When a(n) VMI(s) is running on a node that does not have a running `virt-handler-*` those VMIs are considered orphaned. Meaning that they are no longer able to be managed.
+
+## Diagnosis
+
+You can confirm the alert by finding which nodes your virt-hanlder pods are running on.
+
+```bash
+kubectl -n kubevirt get pods -o wide -l kubevirt.io=virt-handler
+```
+*Output:*
+```
+NAME                 READY   STATUS    RESTARTS   AGE  IP               NODE     NOMINATED NODE   READINESS GATES
+virt-handler-vhqsp   1/1     Running   0          4h   10.244.140.80    node02   <none>           <none>
+virt-handler-xd8jc   1/1     Running   0          4h   10.244.196.168   node01   <none>           <none>
+```
+
+Then check to see which nodes the VMI(s) are running on. If they are running on a node that a virt-handler pod does not exist on those VMIs are orphaned.
+
+```bash
+kubectl get vmis --all-namespaces
+```
+
+*Output:*
+```
+NAMESPACE   NAME            AGE   PHASE       IP    NODENAME
+default     vmi-ephemeral   4s    Scheduled         node02
+```
+
+## Mitigation
+
+Check to see if the DaemonSet that controls the `virt-handler-*` pods is healthy.
+
+```bash
+kubectl -n kubevirt get daemonset virt-handler
+```
+
+*Output:*
+```
+NAME                    DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR            AGE
+virt-handler            2         2         2       2            2           kubernetes.io/os=linux   4h
+```
+
+The DaemonSet is considered healthy if Desired, Ready, and Available are all the same number.  
+
+
+### Unhealthy virt-handler DaemonSet
+Check the DaemonSets status to see what the issues are with deploying the pods.
+
+`kubectl -n kubevirt describe daemonset virt-handler`
+
+
+Another way to check the status is to get the object and read through its status object.  
+
+`kubectl -n kubevirt get daemonset virt-handler -o yaml`
```

Comment:
`oc get daemonset virt-handler -o json | jq .status`
if you want to show only the status stanza